net-libs/msgraph/msgraph-0.2.3.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org meson
+
+DESCRIPTION="GLib-based library for accessing online serive APIs using MS Graph protocol."
+HOMEPAGE="https://gitlab.gnome.org/GNOME/msgraph"
+
+LICENSE="LGPL-2+"
+SLOT="0/1"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc64 ~riscv ~sparc ~x86"
+
+IUSE="debug gtk-doc +introspection man"
+
+RDEPEND="
+	>=dev-libs/glib-2.28.0:2
+	dev-libs/json-glib
+	net-libs/rest
```

Comment:
I would also append :1.0 slot as we don't know if, in the future, a new incompatible slot will appear. It already happened in the past and it's the reason that other reverse deps are specifying the slot too.